src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -351,6 +351,20 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
     this.reset()
   }
 
+  /**
+   * Returns the computed action for the specified org/person compared with their original state.
+   * @param person the person to compare
+   * @returns the action (or null if none)
+   */
+  private computeAction (person: OrgPersonIF): ActionTypes {
+    if (!person) return ActionTypes.REMOVED
+    const original = this.originalParties.find(x => x.officer.id === person.officer.id)
+    if (!original) return ActionTypes.ADDED
+    // ignore "action" when comparing
+    if (!this.isSame(person, original, 'action')) return ActionTypes.EDITED
```

Comment:
Note: I am using `isSame()` and ignoring the action (since one object may have a current action and the other object may not).